ui/src/api/vendors/vendor.js

```diff
@@ -0,0 +1,104 @@
+import { http, apiConfig } from '../base';
```

Comment:
we did renaming proposed above, you will have `/api/acquisitions/vendor.js`

Reply:
Renamed them as proposed.